src/EurekaClient.js

```diff
@@ -1,9 +1,10 @@
 import request from 'request';
 import fs from 'fs';
 import yaml from 'js-yaml';
-import merge from 'deepmerge';
+import merge from 'lodash/merge';
```

Comment:
I don't if we want to bring in such a large dependency when we are only using such a small subset of it's functionality.